src/store/Workspaces/index.ts

```diff
@@ -253,7 +267,7 @@ export const actionCreators: ActionCreators = {
       dispatch({ type: 'UPDATE_WORKSPACE', workspace });
     } catch (e) {
       dispatch({ type: 'RECEIVE_ERROR' });
-      throw new Error(`Failed to start the workspace, ID: ${workspaceId}, ` + e);
+      throw new Error(`Failed to start the workspace, ID: ${workspaceId}, ${e.message ? e.message : ''}`);
```

Comment:
Just curious what problem did you fix here?

Reply:
This is due to the transition to use restApiClient. It is needed for showing a real message instead of ...[object][object].